css-two-column-layout/styles.css

```diff
@@ -0,0 +1,88 @@
+body {
+  font-family: "Karla", sans-serif;
+}
+
+h1 {
+  text-transform: uppercase;
+  letter-spacing: 0.1em;
+}
+
+p {
+  font-size: small;
+  font-weight: lighter;
+}
+
+h3 {
+  text-transform: uppercase;
+  font-size: small;
+}
+
+.header {
+  text-align: center;
+  padding-top: 2rem;
+  padding-right: 7rem;
+  padding-left: 7rem;
+  padding-bottom: 1rem;
+}
+
+.row {
+  display: flex;
+
+  /* padding-left: 2rem;
+  padding-right: 2rem; */
+}
+
+.column-half {
+  display: flex;
+  padding-left: 3rem;
+  padding-right: 3rem;
+  padding-bottom: 2rem;
+  padding-top: 2rem;
+}
+
+.column-half > img {
+  position: relative;
+  width: 25%;
+}
+
+#software {
+  bottom: 50px;
+  right: 5px;
+}
+
+#practice {
+  bottom: 50px;
+  right: 5px;
+}
+
+#circumstance {
+  bottom: 40px;
+  right: 5px;
+}
+
+#data {
+  bottom: 40px;
+  right: 5px;
+}
```

Comment:
Why not make this into two classes you can reuse instead of four individual id selectors?